generated/nise/__init__.py

```diff
@@ -1,6 +1,9 @@
 # -*- coding: utf-8 -*-
 # This file was generated
 
+
+__version__ = '0.2.1.dev0'
```

Comment:
I feel I'm missing what changed to cause this to show up?

Reply:
`nise` no longer has specialized templates for `__init__.py` and `session.py`. This wasn't added to the `nise` template like it should have been. Basically I didn't realize that there was a specialized template.